src/views/credentials/viewCredentialsList.tsx

```diff
@@ -35,7 +36,7 @@ import { SimpleDropdown } from '../../components/simpleDropdown/simpleDropdown';
 import { API_CREDS_LIST_QUERY, API_DATA_SOURCE_TYPES, API_QUERY_TYPES } from '../../constants/apiConstants';
 import { helpers } from '../../helpers';
 import { useAlerts } from '../../hooks/useAlerts';
-import { useAddCredentialApi, useDeleteCredentialApi, useEditCredentialApi } from '../../hooks/useCredentialApi';
+import { useDeleteCredentialApi, useAddCredentialApi, useEditCredentialApi } from '../../hooks/useCredentialApi';
```

Comment:
Since I already added nitpicky comment, here's next one:

why switch order of imports? It used to be alphabetical, which is probably a good default.

Obviously this is not blocking.